Sources/PackageLoading/ModuleMapGeneration.swift

```diff
@@ -24,6 +24,24 @@ extension CModule {
 
 extension ClangModule {
     
+    /// A link-declaration specifies a library or framework
+    /// against which a program should be linked.
+    /// More info: http://clang.llvm.org/docs/Modules.html#link-declaration
+    public enum ModuleMapLinkDecl {
+        case library
+        case framework
+
+        /// Link declaration flag to be used in modulemap.
+        var flag: String {
+            switch self {
+            case library:
+                return "link"
+            case framework:
+                return "link framework"
```

Comment:
Is this required, or is "link framework" the default when auto linking is enabled?